eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/transform/JAXBTransformProvider.java

```diff
@@ -254,5 +270,15 @@ private boolean removePackages() {
             return changed;
         }
 
+        private boolean showNewControls() {
+            boolean isVersion21OrHigher = true;
```

Comment:
I would prefer to not use this intermediate variable and return !(version.startsWith("2.0") || version.startsWith("1")) directly.
or keeping the if/else clerarer
if (version.startsWith("2.0") || version.startsWith("1")) {
               return false;
          } else {
return true;
}